kilombo/service/ncbi.py

```diff
@@ -1,48 +1,51 @@
-import concurrent.futures
 import logging
 import time
 
 import requests
 import xmltodict
 
-logging.basicConfig(level=logging.INFO)
-log = logging.getLogger(__name__)
-
-NCBI_API_KEY = "ed06bd0f3c27a605d87e51e94eecab115908"
+NCBI_API_KEYS = ["ed06bd0f3c27a605d87e51e94eecab115908", "b81884ffa1519f17cae15f6bd21ac8070108"]
```

Comment:
Inyectarlas como variables de entorno mediante el fichero `.env`. 

Más info: https://dev.to/jakewitcher/using-env-files-for-environment-variables-in-python-applications-55a1

Reply:
sí, esto está pendiente, tengo una tarea para ello en el trello
gracias por el recurso!!